wolfcrypt/src/sp_c64.c

```diff
@@ -900,7 +900,7 @@ SP_NOINLINE static void sp_2048_mul_d_36(sp_digit* r, const sp_digit* a,
 #else
     int128_t tb = b;
     int128_t t = 0;
-    sp_digit t2;
+    int128_t t2;
```

Comment:
Don't need this to be a double width type.
Compiler should figure out it doesn't need the top half but better to set the smallest type necessary.

Reply:
Sean this can from the latest master in scripts repo. I’ll revert in the Pr and perhaps you can sync up the script? Thanks @SparkiDev